backend/src/test/java/com/yagubogu/checkin/CheckInIntegrationTest.java

```diff
@@ -143,4 +144,41 @@ void findCheckInHistory_notFoundMember() {
                 .then().log().all()
                 .statusCode(404);
     }
+
+    @DisplayName("인증 여부를 조회한다")
+    @Test
+    void findCheckInStatus() {
+        // given
+        boolean expected = true;
+
+        // when
+        CheckInStatusResponse actual = RestAssured.given().log().all()
+                .contentType(ContentType.JSON)
+                .pathParam("memberId", 1L)
+                .queryParam("date", "2025-07-21")
+                .when().get("/api/check-ins/status/members/{memberId}")
+                .then().log().all()
+                .statusCode(200)
+                .extract()
+                .as(CheckInStatusResponse.class);
+
+        // then
+        assertThat(actual.isCheckIn()).isEqualTo(expected);
```

Comment:
A: true인지만 검사하는건 어떤가요? 저한테는 expected가 여기서는 가독성이 떨어지는 느낌인것 같아서요. true라는 값 자체에 의미가 있다고 생각해요.
```suggestion
        assertThat(actual.isCheckIn()).isTrue();
```